testdata/telemetry/SLE-SERVER-SCCHwInfo/sle12sp5-test.json

```diff
@@ -11,4 +11,4 @@
     "cloud_provider": "",
     "mem_total": 4096
   }
-}
+}
```

Comment:
You appear to have an unnecessary change in this file?

Reply:
I think this was because of my VS Code formatter. I used this file to validate my changes, and it ended up formatting it.

Reply:
Just add back in the blank line at the end of the file and save and commit that and it should remove this unnecessary change.

Better not to have unnecessary changes, especially whitespace ones, if possible.